pipeline/clean/opuscleaner/clean-corpus.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Question: Is there a reason this is in bash rather than python?

Reply:
This is a very simple script that just calls CLI tools and does some IO with them, so I didn't see any reason to write it in Python. Also, we don't have a pattern setup of how to write such things in Python efficiently. It shouldn't be hard, but it's just way faster to write this in bash. As a follow up you can try refactoring this later in Python to compare the complexity and the benefits of such scripts.